dockerfile_parse/parser.py

```diff
@@ -66,12 +67,17 @@ def __setitem__(self, key, value):
 
 
 class DockerfileParser(object):
-    def __init__(self, path=None, cache_content=False, env_replace=True, parent_env=None):
+    def __init__(self, path=None,
+                 cache_content=False,
+                 env_replace=True,
+                 parent_env=None,
+                 fileobj=None):
         """
         Initialize path to Dockerfile
         :param path: path to (directory with) Dockerfile
         :param cache_content: cache Dockerfile content inside DockerfileParser
         :param parent_env: python dict of inherited env vars from parent image
+        :param fileobj: file-like object used to cache content of Dockerfile
         """
         path = path or '.'
```

Comment:
Maybe the first thing this method does is raise a `ValueError` if both `path` and `fileobj` are truthy?